### PR TITLE
Add support for Scala Native

### PR DESCRIPTION
**Pull request by ekrich**
This should be complete if I understand the build file correctly.
- Avoids tests for Scala Native
- Supports production version `0.3.9` and `0.4.0-M2`.

Suggestion and reviews are welcomed.

### CODE REVIEW

**Comment by ekrich** [comment]
Well, going to need some excludes for Scala Native in `travis.yml`.

**Comment by SethTisue** [comment]
(not sure in this particular case, but sometimes it's easier to enumerate the combos you want with `include:`, instead of having to `exclude:` the ones you don' want)

**Comment by ekrich** [comment]
Very good idea, thanks. Had enough for the day.

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
-addSbtPlugin("ch.epfl.scala"          % "sbt-scalafix"             % "0.9.5")
-addSbtPlugin("com.eed3si9n"           % "sbt-buildinfo"            % "0.9.0")
-addSbtPlugin("com.jsuereth"           % "sbt-pgp"                  % "1.1.2-1")
-addSbtPlugin("de.heikoseeberger"      % "sbt-header"               % "5.2.0")
+addSbtPlugin("org.scala-js"           % "sbt-scalajs"                   % scalaJSVersion)
+addSbtPlugin("org.portable-scala"     % "sbt-scalajs-crossproject"      % crossVer)
+addSbtPlugin("org.scala-native"       % "sbt-scala-native"              % scalaNativeVersion)
+addSbtPlugin("org.portable-scala"     % "sbt-scala-native-crossproject" % crossVer)
+addSbtPlugin("org.scala-native"       % "sbt-scala-native"              % scalaNativeVersion)
+addSbtPlugin("org.scala-lang.modules" % "sbt-scala-module"              % "2.0.0")
```
Comment (plokhotnyuk) [inline project/plugins.sbt]: duplicated lines for the native plugin

Comment (ekrich) [reply project/plugins.sbt]: Thanks, removed in the next commit.

**Comment by ekrich** [comment]
Ready for review.

**Comment by SethTisue** [comment]
thank you!

**Comment by gabro** [comment]
🎉

@SethTisue is a release scheduled soon? I'm working on a 2.13 metaconfig release to unblock various projects upstream and I'm using scala-collection-compat there.

**Comment by ekrich** [comment]
I could also use a release as well for sconfig. I initially copied the code into sconfig because there was no Scala Native release.

**Comment by SethTisue** [comment]
> is a release scheduled soon?

let's roll: #241

**Comment by ekrich** [comment]
Looks like using the library with Scala Native `0.3.9` and `0.4.0-M2` worked great. https://github.com/ekrich/sconfig/pull/45

**Pull request merged**